private/pkg/oci_image_spdx.go

```diff
@@ -0,0 +1,138 @@
+package main
```

Comment:
can a simple test be added for this? Would make it clearer what the expected output is.

Reply:
I'll do that. nice suggestion.